route/build/src/standalone/model_setup.f90

```diff
@@ -618,7 +632,7 @@ SUBROUTINE init_time(ierr, message)
   ! "Monthly" option: use 2000-01 as template calendar yr/month
   ! "Daily" option:   use 2000-01-01 as template calendar yr/month/day
   select case(lower(trim(restart_write)))
-    case('annual')
```

Comment:
I think either is acceptable here. But, whichever one you prefer is good. Although you should also make sure the documentation and comments are consistent regarding it as well. The comment above for example is wrong unless you change it also.